google/cloud/storage/examples/run_examples_utils.sh

```diff
@@ -216,6 +216,31 @@ run_all_object_examples() {
       "${bucket_name}" "${encrypted_object_name}"
 }
 
+################################################
+# Run the example showing how to make objects public.
+# Globals:
+#   COLOR_*: colorize output messages, defined in colors.sh
+#   EXIT_STATUS: control the final exit status for the program.
+# Arguments:
+#   bucket_name: the name of the bucket to run the examples against.
+# Returns:
+#   None
+################################################
+run_all_public_object_examples() {
+  local bucket_name=$1
+  shift
+
+  local object_name="object-$(date +%s)-${RANDOM}.txt"
+  run_example ./storage_object_samples insert-object \
+      "${bucket_name}" "${object_name}" "a-string-to-serve-as-object-media"
+  run_example ./storage_object_samples make-object-public \
+      "${bucket_name}" "${object_name}"
+  run_example ./storage_object_samples read-object \
```

Comment:
I don't think this exercises the "public" read given the commands are using the same project.